lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
+
+        @include media($mobile) {
```

Comment:
@talbs I believe that 
````scss 
@include span-columns(4); 
````
should be within each breakpoint scope because its relative to the grid settings defined within each breakpoint variable. It is a coincidence that the parameter value is 4 for each variable, but absolute values that are a result of sass processing will be different since $mobile grid has only 4 columns, while $tablet grid has 8 columns, meaning that for mobile grid we want the course cards to occupy the entire grid, while for tablets we want them to occupy only one half of the grid (so that two course cards are presented in a row).